tests/plugins/test_history_at.py

```diff
@@ -0,0 +1,182 @@
+from .test_history import HistoryWebTest
```

Comment:
The filename `test_history_at` does not match the vocabulary elsewhere (version, state, snapshot, ...)